images/origin/system-container/config.json.template

```diff
@@ -70,15 +70,13 @@
 	    "source": "proc"
 	},
 	{
-	    "destination": "/dev",
-	    "type": "tmpfs",
-	    "source": "tmpfs",
-	    "options": [
-		"nosuid",
-		"strictatime",
-		"mode=755",
-		"size=65536k"
-	    ]
+		"destination": "/dev",
+		"type": "bind",
+		"source": "/dev",
+		"options": [
+			"rbind",
+			"rprivate"
```

Comment:
setting: `"noNewPrivileges": false` in config.json removes the selinux issue, but I'm unsure if this something that should be done? 
On the other hand, this is the way systemd runs it's processes, which makes the container more or less equal to the rpm install (when it comes to privs.) @giuseppe